### PR TITLE
feat: Added direct policy attachment in iam-user module

### PR DESCRIPTION
**Pull request by SlavaNL**
## Description
Add ability to assign a list of custom IAM Policies to selected IAM User during creation

## Motivation and Context
Sometimes IAM Users require some elevation of privileges without including other groups. Also some projects should have a single user without a group, for example the `terraform` user with pre-existing policies to deploy resources. Does not make sense to create a group for this purpose, so the ability to add some policies to IAM User is a nice feature.

## Breaking Changes
None

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Comment by cameron-mcdougle-arturoai** [comment]
This would be really helpful! Thank you. You have my vote for merging.

**Comment by joelgaria** [comment]
Please add this functionality

File: modules/iam-user/outputs.tf [hunk 1]
```diff
@@ -149,3 +149,8 @@ output "iam_user_ssh_key_fingerprint" {
   description = "The MD5 message digest of the SSH public key"
   value       = try(aws_iam_user_ssh_key.this[0].fingerprint, "")
 }
+
+output "custom_iam_policy_arns" {
```
Comment (antonbabenko) [inline modules/iam-user/outputs.tf:153]:
```suggestion
output "policy_arns" {
```

The word `custom` is obsolete in this context. Please change the input variable name as well.

**Review by antonbabenko — CHANGES_REQUESTED** [review]
Almost great. Minor comments.

**Review by antonbabenko — CHANGES_REQUESTED** [review]
Let's be consistent and rename `custom_iam_policy_arns` to `policy_arns` everywhere.

**Comment by SlavaNL** [comment]
@antonbabenko Thanks, I finished it

File: examples/iam-user/main.tf [hunk 2]
```diff
@@ -46,3 +46,21 @@ module "iam_user3" {
   create_iam_access_key         = true
   iam_access_key_status         = "Inactive"
 }
+
+###################################################################
+# IAM user with AmazonSNSReadOnlyAccess policy assigned
```
Comment (antonbabenko) [inline examples/iam-user/main.tf:51]:
```suggestion
# IAM user with IAM policy attached
```

**Review by antonbabenko — APPROVED** [review]

**Comment by antonbabenko** [comment]
This PR is included in version 5.28.0 :tada:

**Comment by igor-nikiforov** [comment]
Unfortunately this not working as expected. See https://github.com/terraform-aws-modules/terraform-aws-iam/issues/408

**Pull request merged**